packages/jaeger-ui/package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/preset-typescript": "^7.21.0",
     "@testing-library/jest-dom": "^6.0.0",
     "@testing-library/react": "^14.0.0",
-    "@types/react": "^18.2.33",
+    "@types/react": "^18.2.37",
```

Comment:
This part gets modified too when depandabot tries to upgrade deps even with resolution. Strange why this didn't happen with PR https://github.com/jaegertracing/jaeger-ui/pull/1967